src/client/pages/ListPage.tsx

```diff
@@ -22,20 +23,37 @@ interface ListPageProps {
   modelName: string;
 }
 
+type FieldsAndRels = (Field | ManifestRelation)[];
+
 const ListPage: React.FC<ListPageProps> = ({ moduleName, modelName }) => {
-  const manifest = useManifest().get({ moduleName, modelName });
   const query = useQuery();
-  const [page, setPage] = useState(Number(query.get("page")) || 1);
-  const [amount, setAmount] = useState(Number(query.get("amount")) || 25);
-  const [search, setSearch] = useState(query.get("search") || "");
+  const manifest = useManifest().get({ moduleName, modelName });
+
+  const [ page, setPage ] = useState(Number(query.get("page")) || 1);
+  const [ amount, setAmount ] = useState(Number(query.get("amount")) || 25);
```

Comment:
The amount can be 0? And if it's that the case, is it ok to set it to 25 in that case? Because the `||` operator will fall to the else for any falsey statement, in other case use `??`

Reply:
It can be 0 because someone can manually enter &amount=0 in the url, but as this controls the amount of elements shown in a list, it makes no sense to allow to show 0 elements.